src/qmapshack/mouse/line/IScrOptEditLine.ui

```diff
@@ -235,7 +235,7 @@
         <item>
          <widget class="QToolButton" name="toolAutoRoute">
           <property name="toolTip">
-           <string>Use auto-routing to between points. (Key: A)</string>
+           <string>Use auto-routing to connect points. (Key: A)</string>
```

Comment:
I think the original intention was to write "Use auto-routing **in** between points"

Reply:
Wiil change it to "original intention".